include/deal.II/matrix_free/operators.h

```diff
@@ -305,6 +305,15 @@ namespace MatrixFreeOperators
   class MassOperator : public Base<dim, Number>
   {
   public:
+    /**
+     * Number typedef.
+     */
+    typedef typename Base<dim,Number>::number_type number_type;
```

Comment:
Do we have a policy somewhere on how we call these types? I'm used to naming them `value_type` in functions like this as that is the convention in the vector and other matrix classes. I'm fine with either variant but we need to do it consistently and I think consistency with other matrices is a good reason.